frontend/amundsen_application/static/js/pages/SearchPage/SearchPanel/styles.scss

```diff
@@ -21,7 +21,7 @@
 
   .checkbox,
   .radio {
-    margin-bottom: $spacer-1;
-    margin-top: $spacer-1;
+    margin-bottom: $spacer-half;
+    margin-top: $spacer-half;
```

Comment:
nit: I kind of like having the little more space between the title and the first checkbox since the text sizes are similar it helps give a little more separation, you could add `margin-top: $spacer-1` for the first child if you want to try that